### PR TITLE
parser: allow alias and const not to be keywords

### PR DESCRIPTION
**Pull request by MatthieuDartiailh**
This implements the solution 2 outlines in #380. Except for the need to manually filter out possibly wrong nodes this looks reasonably clean to me.

Existing tests pass but we probably need some extra ones.

### CODE REVIEW

File: enaml/core/parser/base_parser.py [hunk 1]
```diff
@@ -495,125 +496,104 @@ def p_pragma_arg3(self, p):
...
+        # Storage expression
+        else:
+            self._validate_storage_expr(kind, lineno, p.lexer.lexer)
+            node = enaml_ast.StorageExpr()
+            node.lineno = lineno
+            node.kind = kind
+            node.name = p[2]
+            p[0] = node
+
+    def p_sstorage_alias_const_expr2(self, p):
```
Comment (frmdstryr) [inline enaml/core/parser/base_parser.py]: extra s in storage

Comment (MatthieuDartiailh) [reply enaml/core/parser/base_parser.py]: Good catch and proof we need more tests !!!

**Comment by MatthieuDartiailh** [comment]
The one failure we have is under Python 2 and since I won't make a bugfix release for that it will be fine since we are dropping Python 2. I will try to merge the cppy PR soon to make that clear.

**Comment by frmdstryr** [comment]
I'd rather have python 2 support than alias and const not being keywords.

**Comment by MatthieuDartiailh** [comment]
I won't support Python 2 after january 2020 and I don't plan a release between now and then.

**Comment by MatthieuDartiailh** [comment]
This is ready to be merged from my point of view and will be next week if nobody complains before then.

**Pull request merged**